README.md

```diff
@@ -11,4 +11,52 @@ Visual Studio 2022
+
+[Download Free Trial](https://www.syncfusion.com/downloads/aspnetmvc?utm_source=github&utm_medium=listing&utm_campaign=aspnet-mvc-toogle-switch-button-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/products/aspnetmvc?utm_source=github&utm_medium=listing&utm_campaign=aspnet-mvc-toogle-switch-button-github-samples)
+
+[Documentation](https://ej2.syncfusion.com/aspnetmvc/documentation/switch/getting-started?utm_source=github&utm_medium=listing&utm_campaign=aspnet-mvc-toogle-switch-button-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/asp-net-mvc-theme-switch?utm_source=github&utm_medium=listing&utm_campaign=aspnet-mvc-toogle-switch-button-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/aspnetmvc-ui-components?utm_source=github&utm_medium=listing&utm_campaign=aspnet-mvc-toogle-switch-button-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/aspnetmvc?utm_source=github&utm_medium=listing&utm_campaign=aspnet-mvc-toogle-switch-button-github-samples)
+
+[Online example](https://ej2.syncfusion.com/aspnetmvc/Button/Switch#/material?utm_source=github&utm_medium=listing&utm_campaign=aspnet-mvc-toogle-switch-button-github-samples)
+
+[Knowledge base](https://www.syncfusion.com/kb/aspnetmvc-ui-components?utm_source=github&utm_medium=listing&utm_campaign=aspnet-mvc-toogle-switch-button-github-samples)
+
+
+## About Syncfusion ASP.NET MVC UI Components
+
+Syncfusion's [ASP.NET MVC UI Components](https://www.syncfusion.com/aspnetmvc-ui-components?utm_source=github&utm_medium=listing&utm_campaign=aspnet-mvc-toogle-switch-button-github-samples) library is the only suite that you will ever need to build an application since it contains over 75 high-performance, lightweight, modular, and responsive UI Components in a single package. In addition to Toggle Switch Button, we provide popular ASP.NET MVC Components such as [DataGrid](https://www.syncfusion.com/aspnet-mvc-ui-controls/grid?utm_source=github&utm_medium=listing&utm_campaign=aspnet-mvc-toogle-switch-button-github-samples), [Charts](https://www.syncfusion.com/aspnet-mvc-ui-controls/charts?utm_source=github&utm_medium=listing&utm_campaign=aspnet-mvc-toogle-switch-button-github-samples), [Scheduler](https://www.syncfusion.com/aspnet-mvc-ui-controls/scheduler?utm_source=github&utm_medium=listing&utm_campaign=aspnet-mvc-toogle-switch-button-github-samples), [Diagram](https://www.syncfusion.com/aspnet-mvc-ui-controls/diagram?utm_source=github&utm_medium=listing&utm_campaign=aspnet-mvc-toogle-switch-button-github-samples), and [Word Processor](https://www.syncfusion.com/aspnet-mvc-ui-controls/word-processor?utm_source=github&utm_medium=listing&utm_campaign=aspnet-mvc-toogle-switch-button-github-samples).
```

Comment:
it contains over 80 high-performance......

Reply:
Resolved